libs/partners/mongodb/langchain_mongodb/utils.py

```diff
@@ -15,8 +15,6 @@
 from typing import Any, Dict, List, Union
 
 import numpy as np
-from bson import ObjectId
```

Comment:
could only import if type checking and then not have to update func signatures

Reply:
sphinx still runs type checking blocks

Reply:
(at least this is what I was told. If they don't run in sphinx, agreed this would be better)